R/RecallByManufactuer.R

```diff
@@ -0,0 +1,68 @@
+#ua <- user_agent("http://github.com/WrathSmith/caRecall")
+#ua
+#' get_recall_by_number
+#'
+#' @param manufacturer: A car manufacturer name (ie. Honda, Nissan, etc.)
+#' limit: Number of records to return (default is 25)
+#' page: the serach page result to return (default is 1)
+#'
+#'
+#' @return text output
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' get_recall_by_number(1977044)
+#' }
+recall_manufacturer <- function(manufacturer, limit=25, page=1) {
+
```

Comment:
is there a reason we're using a `limit=25` and `page=1`?

Reply:
This was from the defaults given when testing the data base queries off the GC API store website.  I wasn't sure if they were just for their tests or not, but I thought having a default would be a good idea.

Reply:
This is something we'll need to figure out. I would think that a user would want as much data as possible in terms of the limit default (but also worth giving them option in the function call). Probably need to play around with calls and see what makes sense. There's a comment at the end of the httr docs about pagination as well.